salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -165,8 +166,25 @@
 							
 						}
 					},
-					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ] // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
-				};
+
+					// configuration for toolbar buttons, must conform to `STANDARD_MAPPING`,
```

Comment:
yes, 

![](https://cloud.githubusercontent.com/assets/3933654/26683357/d5743400-46e3-11e7-91b3-d5c92bb63d00.png)

that's respectively `StandoffBoldTag`, `StandoffItalicTag`, `StandoffUnderlineTag`, `StandoffStrikethroughTag`, `StandoffSubscriptTag`, `StandoffSuperscriptTag` and `StandoffUriTag`